modules/rhcos-enabling-multipath.adoc

```diff
@@ -45,53 +45,60 @@ The following procedure enables multipath at installation time and appends kerne
 ----
 $ mpathconf --enable && systemctl start multipathd.service
 ----
-** Optional: If booting the PXE or ISO, you can instead enable multipath by adding `rd.multipath=default` from the kernel command line.
++
+.. Optional: If booting the PXE or ISO, you can instead enable multipath by adding `rd.multipath=default` from the kernel command line.
 
 . Append the kernel arguments by invoking the `coreos-installer` program:
 +
 * If there is only one multipath device connected to the machine, it should be available at path `/dev/mapper/mpatha`. For example:
 +
+ifndef::restricted[]
 [source,terminal]
 ----
-ifndef::restricted[]
-$ coreos-installer install /dev/mapper/mpatha \// <1>
+$ coreos-installer install /dev/mapper/mpatha \//
 --ignition-url=http://host/worker.ign \
 --append-karg rd.multipath=default \
 --append-karg root=/dev/disk/by-label/dm-mpath-root \
 --append-karg rw
+----
 endif::[]
 ifdef::restricted[]
-$ coreos-installer install /dev/mapper/mpatha \// <1>
+[source,terminal]
+----
+$ coreos-installer install /dev/mapper/mpatha \//
 --ignition-url=http://host/worker.ign \
 --append-karg rd.multipath=default \
 --append-karg root=/dev/disk/by-label/dm-mpath-root \
 --append-karg rw \
 --offline
-endif::[]
 ----
-<1> Indicates the path of the single multipathed device.
+endif::[]
+* `/dev/mapper/mpatha`: Indicates the path of the single multipathed device.
```

Comment:
Double-check the formatting here:
<img width="726" height="271" alt="Screenshot 2025-11-14 at 2 28 42 PM" src="https://github.com/user-attachments/assets/8e313fa6-f5ec-49c4-9149-1e9752e64e8c" />